src/components/timeline/Selection.js

```diff
@@ -217,32 +229,43 @@ class TimelineRulerAndSelection extends React.PureComponent<Props, State> {
       this._uninstallMoveAndClickHandlers();
     };
 
-    this._installMoveAndClickHandlers(mouseMoveHandler, clickHandler);
+    this._installMoveAndClickHandlers(
+      mouseStartHandler,
+      mouseMoveHandler,
+      mouseStartHandler
+    );
   };
 
   _installMoveAndClickHandlers(
-    mouseMoveHandler: MouseHandler,
-    mouseClickHandler: MouseHandler
+    mouseStartHandler: PointerHandler,
+    mouseMoveHandler: PointerHandler,
+    mouseEndHandler: PointerHandler
   ) {
     // Unregister any leftover old handlers, in case we didn't get a click for the previous
     // drag (e.g. when tab switching during a drag, or when ctrl+clicking on macOS).
     this._uninstallMoveAndClickHandlers();
 
-    this._handlers = { mouseMoveHandler, mouseClickHandler };
-    window.addEventListener('mousemove', mouseMoveHandler, true);
-    window.addEventListener('click', mouseClickHandler, true);
+    this._handlers = { mouseStartHandler, mouseMoveHandler, mouseEndHandler };
+    window.addEventListener('pointerdown', mouseStartHandler, true);
+    window.addEventListener('pointermove', mouseMoveHandler, true);
+    window.addEventListener('pointerup', mouseEndHandler, true);
   }
 
   _uninstallMoveAndClickHandlers() {
     if (this._handlers) {
-      const { mouseMoveHandler, mouseClickHandler } = this._handlers;
-      window.removeEventListener('mousemove', mouseMoveHandler, true);
-      window.removeEventListener('click', mouseClickHandler, true);
+      const { mouseStartHandler, mouseMoveHandler, mouseEndHandler } =
+        this._handlers;
+      window.addEventListener('pointerdown', mouseStartHandler, true);
+      window.addEventListener('pointermove', mouseMoveHandler, true);
+      window.addEventListener('pointerup', mouseEndHandler, true);
       this._handlers = null;
     }
   }
 
-  _onMouseMove = (event: SyntheticMouseEvent<>) => {
+  _onMouseMove = (event: SyntheticPointerEvent<>) => {
+    if (!event.isPrimary) {
+      return;
+    }
```

Comment:
Maybe add a comment explaining when this can happen? My understanding is that this is useful when the user uses more than 1 finger, is that right?

Reply:
yes